<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TravelAgencyUI</class>
 <widget class="QMainWindow" name="TravelAgencyUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TravelAgencyUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>391</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Kund*in</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QLabel" name="label_8">
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="kundeID">
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="splitter_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QLabel" name="label_9">
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string>Vorname</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="kundeVorname">
          <property name="maximumSize">
           <size>
            <width>74</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="splitter_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QLabel" name="label_11">
          <property name="maximumSize">
           <size>
            <width>43</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string>Nachname</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="kundeNachname">
          <property name="maximumSize">
           <size>
            <width>74</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_10">
           <property name="maximumSize">
            <size>
             <width>41</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>Reisen</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QTableWidget" name="reiseTable">
           <property name="showGrid">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string>Reise Id</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Begin der Reise</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Ende der Reise</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>391</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Reise</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>391</width>
       <height>251</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item>
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_12">
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string>Reise-ID</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="travelIdui">
         <property name="maximumSize">
          <size>
           <width>62</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Buchungen</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QTableWidget" name="buchungTable">
          <property name="maximumSize">
           <size>
            <width>331</width>
            <height>192</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>Buchungstyp</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Start</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Ende</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Preis</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>550</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>550</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Check</string>
    </property>
   </widget>
   <widget class="QPushButton" name="configurationButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>550</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Configuration</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>438</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionEinlesen"/>
    <addaction name="actionIata_Codes"/>
   </widget>
   <widget class="QMenu" name="menukunde">
    <property name="title">
     <string>kunde</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menukunde"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEinlesen"/>
   <addaction name="actionsearch"/>
  </widget>
  <action name="actionEinlesen">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>fileIcon2.png</normaloff>fileIcon2.png</iconset>
   </property>
   <property name="text">
    <string>Einlesen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionsearch">
   <property name="icon">
    <iconset resource="iconPrak3.qrc">
     <normaloff>:/bookingIcons/searchIcon.png</normaloff>:/bookingIcons/searchIcon.png</iconset>
   </property>
   <property name="text">
    <string>search</string>
   </property>
  </action>
  <action name="actionIata_Codes">
   <property name="text">
    <string>Read IATA Codes</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="iconPrak3.qrc"/>
 </resources>
 <connections/>
</ui>
